robot/Cumulus/resources/CustomizableRollupsPageObject.py

```diff
@@ -52,6 +54,68 @@ def create_new_rollup_setting(self, **kwargs):
 			new_button = self.selenium.get_webelement(locator)
 			self.selenium.click_element(new_button)
 			self.selenium.wait_until_page_contains_element(select_locator)
-			self.npsp.populate_modal_form(**kwargs)
+			self.populate_crlp_form(**kwargs)
 			self.selenium.click_button("Save")
 			self.selenium.wait_until_element_is_not_visible(success_toast)
+
+	@capture_screenshot_on_error
+	def clone_rollup(self,rollup_name, **kwargs):
+		""" If new rollup doesn't exist - Clone an existing rollup, enter arguments passed  and create new rollup
+			If new rollup exists - logs that rollup already exists
+		"""
+		current_rollup=self._check_rollup_status(rollup_name)
+		new_label=kwargs['Target Object']+": "+kwargs['Target Field']
+		new_rollup=self._check_rollup_status(new_label)
+		if current_rollup and not new_rollup:
```

Comment:
this crlp is one of those that comes when u enable crlps so it should always exist. If it doesn't exist then clone will fail (I feel we need to raise exception if one of the expected rollup is not present) but need to check this with Rick

Reply:
Agree let's get it confirmed from Rick on the behavior.

Reply:
@skristem @rjanjanam Yes, you can 'clone' what comes 'out of the box' with CRLPs.  I am assuming that we are referring to the **Contact: Total Gifts Two Years Ago** CRLP and cloning that to create the custom CRLP **Total Gifts Three Years Ago**.  Is that easier to do overall than automating from scratch?  There still will be edits you need to make with the cloned **Total Gifts Two Years Ago** CRLP in order for it to conform to the new custom **Total Gifts Three Years Ago** CRLP.

The **Contact: Total Gifts Two Years Ago** out-of-box CRLP should **always be present** when CRLPs are enabled.  Are you seeing differently?  If that does not exist when CRLPs is enabled, then we need to definitely investigate that.  I just created a scratch org and enabled CRLP.  I confirmed that the **Contact: Total Gifts Two Years Ago** was indeed there and I was able to clone it.  

Let me know if I am off base here or if I can be of further help in answering questions.

Reply:
yes that Contact: Total Gifts Two Years Ago clrp is existing and I am using it to clone as ratna already has a test case to create from scratch. This way we will be covering both create from scratch and clone. Also, just confirmed with Rick on the call that we definetly need to fail the test(to notify us) in case Contact: Total Gifts Two Years Ago doesn't exist in the org. So keeping the code as it is.